azure-pipelines.yml

```diff
@@ -139,6 +139,67 @@ jobs:
           echo "##vso[task.setvariable variable=pkg_version]$pkg_version"
         displayName: 'create deb package'
 
+      - script: |
+          sudo apt-get install jq
+        displayName: 'install jq'
+
+      - script: |
+          curl https://packagecloud.io/install/repositories/iofog/iofogctl/script.deb.sh | sudo bash
+          sudo apt-get install iofogctl=2.0.1
```

Comment:
The version must be read from `gradle` file, if we want to match agent and iofogctl versions. Otherwise set it as pipeline variable

Reply:
Updated the script to read version from pipeline